### ISSUE TITLE
Why don't you set public and private key after creating the pushService?

### ISSUE THREAD

**Issue by AhmadAlhajKarim** [web-push-libs/webpush-java#164]
pushService.setPrivateKey(privateKey);
pushService.setPublicKey(publicKey);
This made my code working
Also don't forget to convert the payload to json string

**Comment by martijndwars** [comment]
Thanks for the feedback! Just to see if I understand you correctly; are you saying the documentation/example is unclear/misses these calls? If yes, can you point me to the documentation (wiki/readme)?

**Comment by AhmadAlhajKarim** [comment]
Yes, in the documentation of the usage example is not been mentioned where to use the private and the public key. Missing calls is _after initializing_ the pushService.
//Instantiate the push service, no need to use an API key for Push API
`  pushService = new PushService();
`
**//set the pivate and the public key**
`pushService.setPrivateKey(privateKey);
`
`pushService.setPublicKey(publicKey);
`
// Send the notification
`pushService.send(notification);`

Thank you!

**Comment by newk5** [comment]
Thank you, I was having the same issues and the documentation misslead me too.

I noticed on the java side those calls were missing just like you said, but on the client side it's missing the public key too!
This javascript code:

```javascript
serviceWorkerRegistration.pushManager.subscribe({userVisibleOnly: true})
```
Should be:
```javascript
serviceWorkerRegistration.pushManager.subscribe({userVisibleOnly: true,  applicationServerKey: '<publicKey here>'})
```
